person.rb

```diff
@@ -22,6 +23,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] @fatmahussein Great work adding the add_rental method, however for it to implement the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental, we need to ensure the pass in `person itself, book, and date` as parameters when creating a new rental.

Reply:
Thanks @danmainah